### DISCUSSION TITLE
Localisation system?

### DISCUSSION THREAD

**Discussion by TheTechRobo** [OpenShopChannel/WSC-Web#5] [Ideas]
How are you thinking to do localisations, if you have an idea? Since it's Flask/Python, gettext might be good, though it might not be the best for these needs. I haven't heard much about any other decent ones.

Thoughts?

**Comment by dhtdht020** [comment]
We are going to use Flask-Babel.

**Reply by TheTechRobo** [reply]
Ah, interesting! 

I just looked it up and it definitely seems good.
